test/api/triggerBuildTests.js

```diff
@@ -0,0 +1,29 @@
+const proxyquire = require('proxyquire');
+const sinon = require("sinon");
+const chai = require("chai");
+const assert = chai.assert;
+
+const auth = {
+  url: "test",
+  user: "user",
+  password: "password"
+};
+
+
+const JenkinsWrapper = {
+    job: { build: sinon.stub().yields(null, {}) }
+};
+
+const jenkinsStub = () =>
+     JenkinsWrapper;
+
+var triggerBuild = proxyquire("../../lib/api/triggerBuild", { 'jenkins': jenkinsStub });
+
+describe('triggerBuild', function() {
+  it('it should trigger build', function () {
+    triggerBuild(auth, "job", function (err, data) {
+      assert.isNull(err, "Should not return error")
+      assert.isNotNull(data, "Should return data");
```

Comment:
Here happens the opposite. If data happens to be `undefined` this test will pass. It should be `.isOk()`.